force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -226,23 +298,36 @@ public with sharing class ClassesObjectsMethods {
      * where you need to manipulate and manage book information.
      */
     public class Book {
+        
         // Declare three public instance variables - title, author, pages
         // title
         // author
         // pages
+         public String title { get; set; }
+         public String author { get; set; }
+         public Integer pages { get; set; }
```

Comment:
When your class properties are public there is no need to add the getters and setters to them. I would use this syntax if I wanted to make my setter private, for example. 

```
public String author {get; private set;}
```